src/components/ProofOfDeliveryModal.vue

```diff
@@ -0,0 +1,228 @@
+        label-placement="floating"
+        :disabled="isSubmitting || sameAsBilling" 
+        required 
+      />
+
+      <ion-input 
+        v-model="form.idNumber" 
+        :label="translate('ID Number')" 
+        label-placement="floating"
+        :disabled="isSubmitting || sameAsBilling" 
+        required 
+      />
+
+      <ion-select 
+        v-model="form.relationToCustomer" 
+        :label="translate('Relation to customer')" 
+        label-placement="floating" 
+        placeholder="Select" 
+        :disabled="isSubmitting" 
+        interface="popover"
+      >
+        <ion-select-option value="Self">{{ translate("Self") }}</ion-select-option>
+        <ion-select-option value="Family">{{ translate("Family") }}</ion-select-option>
+        <ion-select-option value="Friend">{{ translate("Friend") }}</ion-select-option>
+      </ion-select>
+
+      <ion-input 
+        v-model="form.email" 
+        :label="translate('Email')" 
+        label-placement="floating"
+        type="email" 
+        :disabled="isSubmitting" 
+      />
+    </div>
+  </ion-content>
+
+  <!-- Fixed Save Button at Bottom -->
+  <ion-footer>
+    <ion-toolbar>
+      <ion-button expand="block" @click="submitForm" :disabled="isSubmitting || !isFormValid">
+        <ion-spinner v-if="isSubmitting" name="crescent" slot="start" />
+        <span>{{ translate("HANDOVER") }}</span>
+      </ion-button>
+    </ion-toolbar>
+  </ion-footer>
+</template>
+
+<script setup>
+import { ref, onMounted, computed, defineProps } from "vue";
+import { modalController } from "@ionic/vue";
+import {
+  IonButtons,
+  IonButton,
+  IonContent,
+  IonHeader,
+  IonToolbar,
+  IonIcon,
+  IonTitle,
+  IonItem,
+  IonLabel,
+  IonInput,
+  IonFooter,
+  IonSelect,
+  IonSelectOption,
+  IonCheckbox,
+  IonSpinner
+} from "@ionic/vue";
+import { closeOutline } from "ionicons/icons";
+import logger from "@/logger";
+import { showToast } from "@/utils";
+import { translate } from "@hotwax/dxp-components";
+import { OrderService } from "@/services/OrderService";
+
+const props = defineProps({
+  order: { type: Object, required: true }
+});
+
+const isSubmitting = ref(false);
+const sameAsBilling = ref(false);
+
+const form = ref({
+  name: "",
+  idNumber: "",
+  relationToCustomer: "Self",
+  email: ""
+});
+
+const billingDetails = ref({});
+
+const isFormValid = computed(() => {
+  return form.value.name.trim() !== "" &&
+    form.value.idNumber.trim() !== "" &&
+    form.value.relationToCustomer !== "";
+});
+
+const getBillingDetails = async () => {
+  try {
+    if (!props.order?.orderId) return;
+    
+    const resp = await OrderService.getBillingDetails({ orderId: props.order.orderId });
+    billingDetails.value = resp?.data?.billToAddress || {};
+
+    form.value.name = billingDetails.value?.toName || billingDetails.value?.contactName || "";
+    form.value.email = billingDetails.value?.email || billingDetails.value?.toEmail || "";
```

Comment:
```suggestion
    form.value.email = billingDetails.value.email || billingDetails.value.toEmail || "";
```

No optional chaning is required as billingDetails.value will be an object in all cases.

Reply:
Done